src/doctrines/Monoidal.jl

```diff
@@ -217,16 +213,12 @@ FIXME: This signature should extend both `MonoidalCategoryWithDiagonals` and
 supported.
 """
 @signature SymmetricMonoidalCategory(Ob,Hom) => MonoidalCategoryWithBidiagonals(Ob,Hom) begin
-  mcopy(A::Ob)::Hom(A,otimes(A,A))
-  mmerge(A::Ob)::Hom(otimes(A,A),A)
+  mcopy(A::Ob)::(A → (A ⊗ A))
```

Comment:
This looks like an example of the problem I mentioned on Slack: the notation `@op mcopy Δ` should be defined (it is not inherited) but I guess that yields a warning.

Reply:
I can show you when you get in, but this appears to be working as we want it to

Here is the code that I used to test this:
```
using Catlab
using Catlab.Doctrines
import Catlab.Doctrines: Ob, Hom

A = Ob(FreeCartesianCategory, :A)
B = Ob(FreeCartesianCategory, :B)
C = Ob(FreeCartesianCategory, :C)
f = Hom(:f, A,B)
g = Hom(:g, B,C)

f ⋅ g

Δ(A)

A′ = Ob(FreeBiproductCategory, :A)
B′ = Ob(FreeBiproductCategory, :B)
C′ = Ob(FreeBiproductCategory, :C)
f′ = Hom(:f, A′,B′)
g′ = Hom(:g, B′,C′)

f′ ⋅ g′

Δ(A′)

println(typeof(Δ(A)))
println(typeof(Δ(A′)))
```

Reply:
It works, but only because the Unicode notation is omitted in MonoidalCategoryWithBidiagonals, when it should really be there! We can discuss.